ethers-providers/src/lib.rs

```diff
@@ -79,6 +79,14 @@ where
     }
 }
 
+/// Structure used in eth_syncing RPC
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(untagged)]
+pub enum SyncingStatus {
+    IsFalse(bool),
```

Comment:
could you add a note on how this works? not clear to me what `IsFalse(false)` vs `IsFalse(true)` means?

Reply:
I added comments, but `IsFalse(true)` is not possible to happen by eth spec. wdyt about adding an additional struct, first for deserialization and second for returning to library call?

Reply:
oh as in the commit you pushed does not work? yeah I'm OK with adding an intermediate non-public datatype that helps with the desert and then having the "simple" enum.

Reply:
Yea i made mistake with that commit, now it should work as expected. Can you check it again if it is okay, I embedded intermediate type inside function.